impl/src/main/java/com/okta/sdk/impl/ds/DefaultResourceFactory.java

```diff
@@ -33,7 +38,7 @@
 
     private final InternalDataStore dataStore;
 
-    private static final List<String> SUPPORTED_PACKAGES = Arrays.asList("com.okta.sdk.", "com.okta.authn.sdk.");
```

Comment:
@jmelberg-okta one of your comments on the previous review had me thinking.  There should not be any refs to `authn` in this repo if we can help it.